packs/pathfinder-monster-core/island-oni.json

```
@@ -399,22 +399,27 @@
                     }
                 },
                 "description": {
-                    "value": ""
+                    "value": "<p>(plus @UUID[Compendium.pf2e.conditionitems.Off-Guard] for 1 round)</p>"
                 },
                 "publication": {
                     "license": "ORC",
                     "remaster": true,
                     "title": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "Note",
+                        "selector": "thunderbolt-damage",
+                        "text": "plus @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard]{Off-Guard} for 1 round."
+                    }
+                ],
```

```suggestion
                "rules": [],
```
No need for the RE when we have it in the description, and this would otherwise need localized.